tests/test_basic_ops_list.py

```diff
@@ -30,225 +22,6 @@ def test_list_of_dicts() -> None:
     assert c[1].key2 == "value2"
 
 
-def test_pretty_list() -> None:
-    c = OmegaConf.create(["item1", "item2", dict(key3="value3")])
-    expected = """- item1
-- item2
-- key3: value3
-"""
-    assert expected == c.pretty()
-    assert OmegaConf.create(c.pretty()) == c
-
-
-def test_pretty_list_unicode() -> None:
-    c = OmegaConf.create(["item一", "item二", dict(key三="value三")])
-    expected = """- item一
-- item二
-- key三: value三
-"""
-    assert expected == c.pretty()
-    assert OmegaConf.create(c.pretty()) == c
-
-
-def test_pretty_strings_float() -> None:
-    c = OmegaConf.create(["10e2", "1.0", 1.0])
-    expected = """- '10e2'
-- '1.0'
-- 1.0
-"""
-    assert c.pretty() == expected
-
-
-def test_pretty_string_boolean() -> None:
-    for t in _utils.YAML_BOOL_TYPES:
-        print(t)
-        c = OmegaConf.create([t, 1])
-        expected = "- '%s'\n- 1\n" % t
-        assert c.pretty() == expected
-
-
-def test_pretty_string_int() -> None:
-    c = OmegaConf.create(["1", 1])
-    expected = "- '1'\n- 1\n"
-    assert c.pretty() == expected
-
-
-def test_list_get_with_default() -> None:
-    c = OmegaConf.create([None, "???", "found"])
-    assert c.get(0, "default_value") == "default_value"
-    assert c.get(1, "default_value") == "default_value"
-    assert c.get(2, "default_value") == "found"
-
-
-@pytest.mark.parametrize(  # type: ignore
-    "input_, expected, list_key",
-    [
-        pytest.param([1, 2], [1, 2], None, id="simple"),
-        pytest.param(["${1}", 2], [2, 2], None, id="interpolation"),
-        pytest.param(
-            {
-                "defaults": [
-                    {"optimizer": "adam"},
-                    {"dataset": "imagenet"},
-                    {"foo": "${defaults.0.optimizer}_${defaults.1.dataset}"},
-                ]
-            },
-            [{"optimizer": "adam"}, {"dataset": "imagenet"}, {"foo": "adam_imagenet"}],
-            "defaults",
-            id="str_interpolation",
-        ),
-    ],
-)
-def test_iterate_list(input_: Any, expected: Any, list_key: str) -> None:
```

Comment:
You are moving unrelated tests.
always review your own diffs before asking for someone else to review them.